src/customers/password-options.md

```diff
@@ -43,6 +43,8 @@ The customer password options determine the level of security that is used for p
 
    - For **Recovery Link Expiration Period (hours)**, enter the number of hours before the password recovery link expires.
 
+   - For **Enable Autocomplete on login/forgot password forms**, set to `Yes`, if you want the fields in the customer login and forgot password forms to be filled automatically from previous entries
```

Comment:
```suggestion
   - If you want the fields in the customer login and forgot password forms to be filled automatically from previous entries, set **Enable Autocomplete on login/forgot password forms** to `Yes`.
```

Reply:
Hi, @leslietilling. Do you want me replace this text?

Reply:
done